---
layout: bootstrap
title: Jobs
permalink: /jobs/
---

Please note applications will be reviewed on an ongoing basis and therefore the advertised post(s) may be filled before the stated deadline. 


***
#### [Software Engineer - The Samples, Phenotypes and Ontologies team](https://www.embl.de/jobs/searchjobs/index.php?ref=EBI_01229)
*An EMBL-EBI contract*
Closing date: 24th June 2018

The Samples, Phenotypes and Ontologies team are looking for an experienced software developer to join their group. You will be working on a project specific to the Open Targets initiative to help develop the infrastructure behind the high-profile GWAS Catalog. The GWAS Catalog is a manually curated, high quality repository of all published genome wide association studies that has been running since 2008. It is considered the “go-to” resource for looking up findings relating human genotype to phenotype associations.

If you’re an experienced developer with an interest in helping solve big biological problems about the genetic basis of disease, this job is perfect for you! Most of the GWAS Catalog infrastructure is written in Java, so you’ll need previous Java experience, but we also have data pipeline services developed in Python and we make use of modern javascript frameworks for our user interfaces. We make good use of tools and frameworks like Spring and Docker to deliver our resources. SPOT is a multitalented team, so you should also be flexible enough to learn new languages to find the right tool for the right problem. You should be enthusiastic about full-stack development, as you’ll be expected to develop, test and deploy tools from database to user interface. You should be keen to advance your knowledge of the latest industry standard toolkits to help continuously innovate the catalog.

In this role, we’ll be asking you to help develop the infrastructure that manages the data in the GWAS Catalog - this means you’ll be building tools and pipelines to capture new data from submitters, and to serve it back to the community through the Open Targets platform, via APIs. Individual GWAS datasets can be very large - tens or hundreds of gigabytes - and we’re storing billions of records, so this requires someone who can get creative to build solutions to these sort of big data problems.

[More details at the EMBL-EBI website](https://www.embl.de/jobs/searchjobs/index.php?ref=EBI_01229)


***
#### [Senior Database Curator - Cancer Genetics](https://jobs.sanger.ac.uk/wd/plsql/wd_portal.show_job?p_web_site_id=1764&p_web_page_id=352145)
*A Wellcome Sanger Institute contract*
Closing date: 24th June 2018

We are seeking a motivated and experienced scientist in cancer informatics to play a crucial role in the expansion and curation of the Cancer Gene Census in the COSMIC team at the Sanger institute, Hinxton. In this project, we aim to describe in detail every cancer-causing gene and how its mutated dysfunction drives disease.

COSMIC (http://cancer.sanger.ac.uk) is working with Open Targets (http://www.targetvalidation.org) to expand the world’s most comprehensive description of the genes causing human cancer. This is an exciting position within the COSMIC team, where you will have the opportunity to explore & interpret a wide range of literature, database and analytic resources to maximise the impact of this high-profile global resource, the Cancer Gene Census.

You will work together with COSMIC scientific team, bringing a range of analytic, curation and literature techniques to describe the oncogenic impact of genes implicated in cancer, including content across broad genomic surveys and model organism experiments. Currently the Census contains over 700 genes with full mechanistic description, together with a complete dysfunction impact assessment of approx. 1/3 of them. Our aim is to expand the Cancer Gene Census with new genes, and to generate full descriptive coverage of all Census genes with evidence of oncogenic activity, with descriptive evidence-based content including: mechanistic processes by which these genes are mutated to drive disease, definition of their impact in every cancer type, and description of oncogenic dysfunction of each gene in terms of the 10 Hallmarks of Cancer.

[More details at the Sanger website](https://jobs.sanger.ac.uk/wd/plsql/wd_portal.show_job?p_web_site_id=1764&p_web_page_id=352145)


***
#### [Bioinformatician - Open Targets Core Team](https://ig14.i-grasp.com/fe/tpl_embl01.asp?newms=jj&id=56579&aid=15470)
*An EMBL-EBI contract*
Closing date: 22 July 2018

Open Targets is a recently established public-private initative between the EBI, GlaxoSmithKline, Biogen, Takeda, Celgene and the Sanger Institute to combine large-scale genomic experiments with objective statistical and computational techniques to identify and validate the causal links between targets, pathways and diseases.

We are looking for an experienced bioinformatician for the core Open Targets platform project which is involved in large-scale biological data analysis and integration for target identification and prioritisation.

The postholder will join the Open Targets team developing the data processing and database infrastructure to provide the core deliverable of the centre, a genome-wide database of target validity (https://www.targetvalidation.org).

Responsibilities will involve development and refinement of suitable data feeds from databases within and outside the partnership including interaction with other groups, establishment of automated processing pipelines and quality control procedures, as well as exploratory data analysis in the area of target prioritisation.

In addition the post will involve extensive interaction with backend database developers to refine and test the data processing pipeline.
The post holder will work with an existing bioinformatician working on the core database, and report to the Open Targets Scientific Director.

[More details at the EMBL website](https://ig14.i-grasp.com/fe/tpl_embl01.asp?newms=jj&id=56579&aid=15470)


***
