---
layout: bootstrap
title: Jobs
permalink: /jobs/
---

Please note applications will be reviewed on an ongoing basis and therefore the advertised post(s) may be filled before the stated deadline. 


***
#### [Senior Bioinformatician - Cancer Dependency Map Analytics](https://jobs.sanger.ac.uk/wd/plsql/wd_portal.show_job?p_web_site_id=1764&p_web_page_id=349708)
*A Wellcome Sanger Institute contract*
Closing date: 17th June 2018

The Wellcome Sanger Institute is seeking for an experienced Bioinformatician to provide computational support to the international Cancer Dependency Map consortium, and to other projects engaged in the analysis of data from genome-editing and functional-genomics screens, in collaboration with Open Targets. The selected candidate will join the Cancer Dependency Map Analytics team, actively interacting with the Cancer Dependency Map consortium, whose broad goal is to identify vulnerabilities and dependencies that could be exploited therapeutically in every cancer cell.

The successful candidate will be able to extensively use computational pipelines for pre-processing and quality control assessment of data from genome-editing screens, and to implement new analytical pipelines on an individual project needs’ basis extending existing software, writing, documenting and maintaining code packages on public/internal repositories. Past experience with genomic data curation and familiarity with the management of data from large-scale in-vitro drug/functional-genomic screens will be positively considered. Finally, the selected candidate will interact with Open Targets partners and collaborators, and with web development teams to coordinate data/results flows on the public domain.

This position offers the opportunity to work at one of the world’s leading genomic centres at the forefront of genomic research. The successful candidate will have access to Sanger's computational resources, including a 15000+ core computational cluster, the largest in life science research in Europe, and multiple petabytes of high-speed cluster file systems.

[More details at the Sanger website](https://jobs.sanger.ac.uk/wd/plsql/wd_portal.show_job?p_web_site_id=1764&p_web_page_id=349708)


***
#### [Postdoctoral fellow (Immune Screening) - Experimental Cancer Genetics](https://jobs.sanger.ac.uk/wd/plsql/wd_portal.show_job?p_web_site_id=1764&p_web_page_id=349679)
*A Wellcome Sanger Institute contract*
Closing date: 17th June 2018

With the completion of the human genome and analysis of variation across multiple individuals, attention is now turning to the functional analysis of genes in mammals and their validation as therapeutic targets. Open Targets aims to provide evidence on the biological validity of therapeutic targets and provide an initial assessment of the likely effectiveness of pharmacological intervention on these targets, using genome-scale experiments and analysis. This pioneering public-private partnership between Biogen, Celgene, EMBL-EBI, GlaxoSmithKline, Takeda, and the Wellcome Sanger Institute, aims to provide a R&D framework that applies to all aspects of human disease, and to share its data openly with the scientific community (http://www.opentargets.org).

A new project funded by Open Targets, joint between the Experimental Cancer Genetics group headed by Dr David Adams at the Wellcome Sanger Institute together with Open Targets partners, is to perform in depth analysis of factors that control T cell survival and differentiation within the tumour microenvironment. This will involve the use of bespoke targeted CRISPR screens applied to primary cells and using in vivo tumour models in order to identify novel candidates genes and/or pathways for validation and in-depth characterization.

We are looking for a talented Postdoctoral Fellow to be responsible for the immune aspect of this project to include the isolation and culture of the T cells required the in vivoscreens and their subsequent isolation from tissues. You will follow the validation and characterisation phases of the project.

[More details at the Sanger website](https://jobs.sanger.ac.uk/wd/plsql/wd_portal.show_job?p_web_site_id=1764&p_web_page_id=349679)


***
#### [Software Engineer - The Samples, Phenotypes and Ontologies team](https://www.embl.de/jobs/searchjobs/index.php?ref=EBI_01229)
*An EMBL-EBI contract*
Closing date: 24th June 2018

The Samples, Phenotypes and Ontologies team are looking for an experienced software developer to join their group. You will be working on a project specific to the Open Targets initiative to help develop the infrastructure behind the high-profile GWAS Catalog. The GWAS Catalog is a manually curated, high quality repository of all published genome wide association studies that has been running since 2008. It is considered the “go-to” resource for looking up findings relating human genotype to phenotype associations.

If you’re an experienced developer with an interest in helping solve big biological problems about the genetic basis of disease, this job is perfect for you! Most of the GWAS Catalog infrastructure is written in Java, so you’ll need previous Java experience, but we also have data pipeline services developed in Python and we make use of modern javascript frameworks for our user interfaces. We make good use of tools and frameworks like Spring and Docker to deliver our resources. SPOT is a multitalented team, so you should also be flexible enough to learn new languages to find the right tool for the right problem. You should be enthusiastic about full-stack development, as you’ll be expected to develop, test and deploy tools from database to user interface. You should be keen to advance your knowledge of the latest industry standard toolkits to help continuously innovate the catalog.

In this role, we’ll be asking you to help develop the infrastructure that manages the data in the GWAS Catalog - this means you’ll be building tools and pipelines to capture new data from submitters, and to serve it back to the community through the Open Targets platform, via APIs. Individual GWAS datasets can be very large - tens or hundreds of gigabytes - and we’re storing billions of records, so this requires someone who can get creative to build solutions to these sort of big data problems.

[More details at the EMBL-EBI website](https://www.embl.de/jobs/searchjobs/index.php?ref=EBI_01229)


***
#### [Senior Database Curator - Cancer Genetics](https://jobs.sanger.ac.uk/wd/plsql/wd_portal.show_job?p_web_site_id=1764&p_web_page_id=352145)
*A Wellcome Sanger Institute contract*
Closing date: 24th June 2018

We are seeking a motivated and experienced scientist in cancer informatics to play a crucial role in the expansion and curation of the Cancer Gene Census in the COSMIC team at the Sanger institute, Hinxton. In this project, we aim to describe in detail every cancer-causing gene and how its mutated dysfunction drives disease.

COSMIC (http://cancer.sanger.ac.uk) is working with Open Targets (http://www.targetvalidation.org) to expand the world’s most comprehensive description of the genes causing human cancer. This is an exciting position within the COSMIC team, where you will have the opportunity to explore & interpret a wide range of literature, database and analytic resources to maximise the impact of this high-profile global resource, the Cancer Gene Census.

You will work together with COSMIC scientific team, bringing a range of analytic, curation and literature techniques to describe the oncogenic impact of genes implicated in cancer, including content across broad genomic surveys and model organism experiments. Currently the Census contains over 700 genes with full mechanistic description, together with a complete dysfunction impact assessment of approx. 1/3 of them. Our aim is to expand the Cancer Gene Census with new genes, and to generate full descriptive coverage of all Census genes with evidence of oncogenic activity, with descriptive evidence-based content including: mechanistic processes by which these genes are mutated to drive disease, definition of their impact in every cancer type, and description of oncogenic dysfunction of each gene in terms of the 10 Hallmarks of Cancer.

[More details at the Sanger website](https://jobs.sanger.ac.uk/wd/plsql/wd_portal.show_job?p_web_site_id=1764&p_web_page_id=352145)


***

