---
layout: page
title: Projects
permalink: /projects/
---

Projects
Open Targets employs large-scale human genetics and genomics data to change the way drug targets are identified and validated. We have established a set of interlinking projects to develop both the data and analytical processes that implicate targets as valid, and the core platform to provide the information to a diverse audience of users. We are developing 2 major areas of work, which in each case can be further subdivided into individual workstreams. The major areas are the core bioinformatics platforms, including pipelines and a database to integrate existing target validation data as well as a public web portal to serve the integrated views, and experimental projects which will generate new data to feed into the database.

We have established a portfolio of experimental projects that we believe will provide target validation information relevant to key therapeutic areas. Our approach is to use high throughput methodologies that can address as near as possible the full range of relevant targets (ideally the whole genome) in systems that are relevant to the physiology of diseases with high unmet need. As such we have identified oncology, inflammatory bowel diseases (IBD), respiratory disease, inflammation and immunity as suitable therapeutic areas because in addition to the pressing need for developments in these areas due to disease burden, there is complementary expertise between the partners’ research interests.

We are using methods that provide cross cutting themes across multiple therapeutic areas. We have chosen to focus on the use of genetics as a tool for target identification and validation, and subsequent exploitation of cellular models of disease through the new gene editing technologies or single cell analysis. We are investigating the use of large populations and approaches such as Mendelian Randomisation including for metabolomic and proteomic data. In addition to clinical samples, iPS cells and cellular organoids are resources that can provide cellular phenotyping at scale with more physiological relevance than transformed cell lines.


Core Bioinformatics and Computational Pipelines
The Core components provide the backbone of information and display for the Target Validation Platform. The key challenge is to integrate data sources that affect the validity of a target in a single infrastructure, allowing seamless interrogation of these data. This integration involves both the more conceptually easy (though often practically far harder) integration of related information, and also the more complex joint computation of information together. A good example would be the integration of rare and common disease genetics signals in a pathway network to indicate how far in the large network of inter-related signalling pathways the genetic evidence supports involvement in the disease. For overall data integration, we span many information types relevant to human disease biology, and our approach is to leverage the power of existing data analysis and curation efforts within the Open Targets partners and beyond.


Oncology focus
Cancer is a massive burden on human health, afflicting 1 out of 3 people and causing 30% of deaths in the UK, and is an area of major opportunity in drug development. A shared theme across the 4 oncology workstreams is the application of genomic data from analysis of clinical samples to guide target development. We are using a variety of accessible cancer resources to curate and analyze clinical genomic datasets to identify driver genes (mutations, amplification, deletions, gene-fusions) across multiple cancer sub-types. This information will enable the identification of putative targets, the selection of model systems that best reflect the biology of tumours, as well as guide the analysis of experimental results to identify clinically relevant associations. To facilitate integration of our results and maximise their utility, priority will be given to tumour types and genomic alterations, such as KRAS-mutated lung and colorectal cancer, which are of unmet clinical need. 


IBD focus
The two common forms of IBD, Crohn’s disease and Ulcerative Colitis, have both been remarkably successful in the gene identification arena (163 associated loci), and demonstrate nearly 2/3 overlap in terms of genetic risk. Furthermore, there is clear unmet need in IBD, as many patients do not respond to anti-TNFs and poor efficacy is reported for existing alternative therapies, such as Humira and Tysabri. The UK has one of the largest IBD cohorts worldwide with 20,000 patients consented for genetic study with DNA at Sanger. Cell types of interest in studying IBD (e.g. by epigenomics) are reasonably accessible, including circulating immune cells, gut epithelium, etc. Although there is a significant amount of human data already available for IBD, there are still large areas that remain relatively unexplored.


Respiratory focus
There are two projects with a respiratory focus. In the first we capitalise on the prior discovery of de novo mutations underlying certain extreme sub-types of bronchiectasis in specific families (Activated PI3K delta Syndrome). The project is sequencing a defined sub-population of bronchiectasis patients likely to have a similar genetic basis. This achieves two goals; firstly it provides a specific sub-population of patients in which the PI3K delta inhibitor is a potential experimental medicine; secondly this may identify other targets in the PI3K pathway associated with the disease.


Inflammation and immunity
Dysregulation of macrophages has been implicated in a range of disease processes in immunoinflammation, respiratory disease, fibrosis and cancer. One of the main macrophage cellular phenotypes is phagocytosis, which can be measured using fluorescein isothiocyanate (FITC)-labeled zymosan particles. The initial work is two-fold; comparison of iPSC derived macrophages compared to macrophages from primary monocytes and the development of an iPS gene editing ready cell line. Next a genome-wide gene editing library will be transfected to identify genes that modulate the phagocytosis phenotype.


Cell line epigenomes
We also aim to develop standard epigenome profiles of cell models in use within the pharmaceutical industry and academia to establish a systematic approach for the determination of human biological and disease relevance through the generation of epigenome data.  We will generate epigenome (RNA-seq, open chromatin, and specific histone modifications) profiles for as many of these as is both appropriate and feasible. These data will be compared with existing reference data in the public domain (ENCODE, Epigenome Roadmap, IHEC) to identify the closest neighbours (by clustering, PCA, etc).

https://vimeo.com/149309356