---
layout: bootstrap
title: Resouces
permalink: /resources/
---

# Research Publications
* [Functional linkage of gene fusions to cancer cell fitness assessed by pharmacological and CRISPR-Cas9 screening](https://www.nature.com/articles/s41467-019-09940-1) Picco G et al. DOI: 10.1038/s41467-019-09940-1 
* [Prioritisation of cancer therapeutic targets using CRISPR-Cas9 screens](https://www.nature.com/articles/s41586-019-1103-9) Behan F et al. DOI: 10.1038/s41586-019-1103-9 
* [The NHGRI-EBI GWAS Catalog of published genome-wide association studies, targeted arrays and summary statistics 2019](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky1120/5184712) Buniello A et al. DOI:10.1093/nar/gky1120
* [ChEMBL: Towards Direct Deposition of Bioassay Data](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky1075/5162468) Mendez D et al. DOI:10.1093/nar/gky1075
* [Open Targets Platform: new developments and updates two years on](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky1133/5193331) Carvalho-Silva D et al. DOI:10.1093/nar/gky1133
* [Complex Portal 2018 - extended content and enhanced visualisation tools](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky1001/5144138) Meldal B et al. DOI:10.1093/nar/gky1001
* [The COSMIC Cancer Gene Census: describing genetic dysfunction across all human cancers](https://rdcu.be/9Zcg) Sondka Z et al. DOI:10.1038/s41568-018-0060-1
* [Uncovering new disease indications for G-protein coupled receptors and their endogenous ligands](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2392-y) Freudenberg J et al. DOI:10.1186/s12859-018-2392-y
* [Unsupervised correction of gene-independent cell responses to CRISPR-Cas9 targeting](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-4989-y) Iorio F et al. DOI:10.1186/s12864-018-4989-y
* [Pathway-based dissection of the genomic heterogeneity of cancer hallmarks’ acquisition with SLAPenrich](https://www.nature.com/articles/s41598-018-25076-6) Iorio F et al. DOI:10.1038/s41598-018-25076-6 
* [Interleukins and their signaling pathways in the Reactome biological pathway database](https://www.ncbi.nlm.nih.gov/pubmed/29378288) Jupe S et al. DOI:10.1016/j.jaci.2017.12.992
* [Designing an intuitive web application for drug discovery scientists](https://doi.org/10.1016/j.drudis.2018.01.032) Karamanis N et al. DOI:10.1016/j.drudis.2018.01.032
* [Comparison, alignment, and synchronization of cell line information between CLO and EFO](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1979-z) Ong E et al. DOI:10.1186/s12859-017-1979-z  
* [Unsupervised correction of gene-independent cell responses to CRISPR-Cas9 targeting](https://www.biorxiv.org/content/early/2017/12/03/228189) Iorio F et al. DOI:10.1101/228189 
* [Expression Atlas: gene and protein expression across multiple studies and organisms](https://academic.oup.com/nar/article/46/D1/D246/4641906) Papatheodorou I et al. DOI:10.1093/nar/gkx1158 
* [Uncovering novel repositioning opportunities using the Open Targets platform](http://www.sciencedirect.com/science/article/pii/S1359644617301897) Khaladkar M et al. DOI:10.1016/j.drudis.2017.09.007
* [In silico prediction of novel therapeutic targets using gene–disease association data](https://translational-medicine.biomedcentral.com/articles/10.1186/s12967-017-1285-6) Ferrero E et al. DOI:10.1186/s12967-017-1285-6
* [Reactome enhanced pathway visualization](https://academic.oup.com/bioinformatics/article/doi/10.1093/bioinformatics/btx441/3930125/Reactome-enhanced-pathway-visualization) Sidiropoulos K et al. DOI:10.1093/bioinformatics/btx441
* [ProtVista: visualization of protein sequence annotations](https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btx120) Watkins X et al. DOI:10.1093/bioinformatics/btx120
* [Reactome pathway analysis: a high-performance in-memory approach](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1559-2) Fabregat A et al. DOI:10.1186/s12859-017-1559-2
* [Open Targets: a platform for therapeutic target identification and validation](https://academic.oup.com/nar/article/45/D1/D985/2605745/Open-Targets-a-platform-for-therapeutic-target) Koscielny G et al. DOI:10.1093/nar/gkw1055
* [Transcription factor activities enhance markers of drug sensitivity in cancer](http://cancerres.aacrjournals.org/content/early/2017/12/09/0008-5472.CAN-17-1679) Garcia-Alonso L et al. DOI:10.1158/0008-5472.CAN-17-1679
* [Fine-mapping inflammatory bowel disease loci to single-variant resolution](https://www.nature.com/articles/nature22969) Huang H et al. DOI:10.1038/nature22969
* [Literature Evidence in Open Targets – a target validation platform](https://jbiomedsem.biomedcentral.com/articles/10.1186/s13326-017-0131-3)  Kafkas S et al. DOI:10.1186/s13326-017-0131-3
* [TnT: A set of libraries for visualising trees and track-based annotations in the web](https://www.ncbi.nlm.nih.gov/pubmed/27153646) Pignatelli M DOI:10.1093/bioinformatics/btw210
* [Warped Matrix Factorisation](https://link.springer.com/chapter/10.1007%2F978-3-319-46227-1_49) Pratanwanich N et al. DOI:10.1007/978-3-319-46227-1_49
* [Webulous and the Webulous Google Add-On--a web service and application for ontology building from templates](https://jbiomedsem.biomedcentral.com/articles/10.1186/s13326-016-0055-3) Jupp S et al. DOI:10.1186/s13326-016-0055-3
* [Linking rare and common disease: mapping clinical disease-phenotypes to ontologies in therapeutic target validation](https://jbiomedsem.biomedcentral.com/articles/10.1186/s13326-016-0051-7) Sarntivijai S et al. DOI:10.1186/s13326-016-0051-7


# Review Articles 
* [Applications of machine learning in drug discovery and development](https://www.nature.com/articles/s41573-019-0024-5) Vamathevan, J et al. DOI: 10.1038/s41573-019-0024-5
* [Ten simple rules for delivering live distance training in bioinformatics across the globe using webinars](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006419) Carvalho-Silva D et al. DOI:10.1371/journal.pcbi.1006419 
* [Computational approaches for interpreting scRNA-seq data](https://febs.onlinelibrary.wiley.com/doi/abs/10.1002/1873-3468.12684) Rostom R et al. DOI:10.1002/1873-3468.12684 
* [Harnessing public domain data to discover and validate therapeutic targets](https://www.tandfonline.com/doi/abs/10.1080/17460441.2017.1329296?journalCode=iedc20) Reisdorf WC et al. DOI:10.1080/17460441.2017.1329296
* [The impact of rare and low-frequency genetic variants in common disease](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1212-4) Bomba L et al. DOI:10.1186/s13059-017-1212-4
* [Use of Biomedical Ontologies for Integration of Biological Knowledge for Learning and Prediction of Adverse Drug Reactions](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5398297/) Zaman S et al. DOI:10.1177/1177625017696075
* [Genetics and immunity in the era of single cell genomics](https://academic.oup.com/hmg/article/25/R2/R141/2198146) Vieira Braga FA et al. DOI:10.1093/hmg/ddw192 
* [Using human genetics to make new medicines](http://europepmc.org/abstract/MED/26370900) Barrett JC et al.  DOI:10.1038/nrg3998
* [Understanding inflammatory bowel disease via immunogenetics](https://www.sciencedirect.com/science/article/pii/S0896841115300147) de Lange KM et al. DOI:10.1016/j.jaut.2015.07.013
* [Strategies for fine-mapping complex traits](https://academic.oup.com/hmg/article/24/R1/R111/671614) Spain SL et al. DOI:10.1093/hmg/ddv260
<br/>
<br/>

# [Open Targets Platform](http://www.targetvalidation.org/)
We have developed the [Open Targets Platform](http://www.targetvalidation.org/) to provide summaries of the evidence for the involvement of a specific gene with a selected disease. 


<br/>
<br/>


# [Open Targets Genetics](https://genetics.opentargets.org/)
[Open Targets Genetics](https://genetics.opentargets.org/) is a portal for investigation of Genome Wide Association Study (GWAS) data to assist in identifying the causal genes underlying each assoication, and hence to prioritise drug targets. The portal aggregates and merges genetic associations curated from literature and newly-derived loci from UK Biobank with (open source) functional genomics data including epigenetics (e.g., chromatin conformation, chromatin interactions) and quantitative trait loci (e.g., eQTLs from GTEX, pQTL), and applies statistical fine-mapping across thousands of trait-associated loci, to resolve association signals and link each variant to its proximal and distal target gene(s), using a single evidence score. 


<br/>
<br/>

# [Project Score](https://score.depmap.sanger.ac.uk)
[Project Score](https://score.depmap.sanger.ac.uk) is a web portal that allows researchers to explore the results of CRISPR-Cas9 whole-genome drop out screens across a diverse collection of human cancer cell models and to identify dependencies in cancer cells to help guide precision cancer medicines.

Please see [Behan et al. 2019](https://www.nature.com/articles/s41586-019-1103-9) for more information. 

<br/>
<br/>

# [epiChoose](http://epichoose.shiny.opentargets.io/epiChoose/)
epiChoose is an application for quantifying the relatedness between cell lines and primary cells. We have extensively profiled a number of commonly used cell line models across a number of tissues. This profiling consists of epigenetic (histone modification, CTCF, ATAC-seq) and transcriptional (RNA-seq) whole-genome measurements. From this data, we have established a platform that provides information on the “distance” between candidate cell models and target primary cells. By ranking and selecting cell line models in a data-driven manner, the best model for the experiment can be chosen, based on epigenetic and transcriptional evidence and not historical usage.


<br/>
<br/>

# [CRISPRcleanR](https://github.com/francescojm/CRISPRcleanR)
Genome editing by CRISPR-Cas9 technology allows large-scale screening of gene essentiality in cancer. A confounding factor when interpreting CRISPR-Cas9 screens is the high false-positive rate in detecting essential genes within copy number amplified regions of the genome. We have developed the computational tool CRISPRcleanR which is capable of identifying and correcting gene-independent responses to CRISPR-Cas9 targeting. CRISPRcleanR uses an unsupervised approach based on the segmentation of single-guide RNA fold change values across the genome, without making any assumption about the copy number status of the targeted genes.
CRISPRcleanR is implemented as an R package and as an interactive Python package with full documentation, tutorials, built in datasets to reproduce the results in this manuscript, and is publically available (R package: https://github.com/francescojm/CRISPRcleanR and Python package: https://github.com/cancerit/pyCRISPRcleanR). The Python implementation is dockerized making it platform independent and usable in cloud environments (https://dockstore.org/containers/quay.io/wtsicgp/dockstore-pycrisprcleanr).

Please see [Iorio et al. 2018](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-4989-y) for more information. 

<br/>
<br/>

# [CELLector (Genomics Guided Selection of Cancer in vitro Models)](http://ot-cellector.shiny.opentargets.io/CELLector_App/)
[CELLector](http://ot-cellector.shiny.opentargets.io/CELLector_App/) is a computational tool implemented in an open source R Shiny application and R package that allows researchers to select the most relevant cancer cell lines in a genomic-guided fashion. CELLector combines methods from graph theory and market basket analysis; it leverages tumour genomics data to explore, rank, and select optimal cell line models in a user-friendly way, enabling scientists to make appropriate and informed choices about model inclusion/exclusion in retrospective analyses and future studies. Additionally, it allows the selection of models within user-defined contexts, for example, by focusing on genomic alterations occurring in biological pathways of interest or considering only predetermined sub-cohorts of cancer patients. Finally, CELLector identifies combinations of molecular alterations underlying disease subtypes currently lacking representative cell lines, providing guidance for the future development of new cancer models. 

Please see [Najgebauer et al. 2018](https://www.biorxiv.org/content/early/2018/03/03/275032) for an explanation of the approach.

<br/>
<br/>

# [LINK (LIterature coNcept Knowledgebase)](https://link.opentargets.io/)
A new tool called [LINK](https://link.opentargets.io/) (LIterature coNcept Knowledgebase) has been developed that allows the exploration of half a billion relations between genes, diseases, drugs and key concepts extracted from PubMed abstracts using NLP (Natural Language Processing).

Once MEDLINE relaxed their license for obtaining and analysing publication data last year, we started looking for novel ways to mine their data. We wanted to exploit the biomedical knowledge often buried in the literature to help scientists generate new hypotheses for the identification of new drug targets.
For this purpose, we have built Library, an open source ecosystem comprising:
* a pipeline that allows us to quickly run a large scale NLP analysis.
* an API that serves the resulting data.
* a user interface to explore this data.

Our pipeline annotates genes, diseases and drugs present in PubMed abstracts, and extracts key concepts. 

<br/>
<br/>

# [DoRothEA (Discriminant Regulon Expression Analysis)](http://dorothea.opentargets.io/#/)
[DoRothEA](http://dorothea.opentargets.io/#/) (Discriminant Regulon Expression Analysis) is a research resource that can be used to search candidate TF-drug interactions in cancer.

Due to their location as downstream effectors of signalling pathways, aberrant activities in upstream driver genes (even if not mutated) will cause altered TF activities, thus proposing TFs as sensors of pathway dysregulation and alternative markers. Here we study the role of 127 TFs in drug sensitivity across ~1,000 cancer cell lines screened with 265 anti-cancer compounds from the GDSC. In our first approach we studied how the TF activity pattern of an individual affects drug response and mined for single TF-drug statistical interactions. In our second approach we screened for TFs whose activity patterns complement or improve well-established genomic markers in the prediction of drug response.

Please see [Garcia-Alonso et al.](http://cancerres.aacrjournals.org/content/early/2017/12/09/0008-5472.CAN-17-1679) for an explanation of the approach.

<br/>
<br/>

# Open Targets Principles

Open Targets is guided by the following principles: 
*   We are focused on pre‐competitive research that will enable the systematic identification and prioritisation of targets
*   We are committed to rapid publication and making data, methods and results publically available as soon as possible
*   We believe in non‐exclusive partnerships that foster the free exchange of ideas and expertise
 
We place in the public domain all our new informatics tools, experimental methods, platforms and the data generated by our projects as soon as is practical. We do not plan to seek patent protection for IP arising from Open Targets. However, we recognise that instances may arise where this is appropriate to support our mission. Therefore, Open Targets has a Joint Patent Committee with scientific and legal experts from all of our partners that reviews all potential publications (but not raw data) prior to submission. 
 
In practice, this means:
*   For the development and execution of Open Targets Research Projects each Member has agreed to license its Background IP to the other members.
*   All Members have agreed to license Open Targets Arising IP to each other for use in Open Targets Research Projects and for the Members’ research and development activities.
*   Any IP that relates directly and solely to an Industry Partner Compound will belong to the Industry Partner.
*   Any other IP arising from an Open Targets Research Project will belong to the Member(s) that invented it.



