---
layout: default
permalink: /resources
title: Resources
---
<header class="py-5" style="background-color:#3489ca">
  <div class="container">
    <div class="row">
      <div class="col-sm-12 text-center text-white">
        <h1>Resources</h1>
      </div>
    </div>
  </div>
</header>

<section class="my-5 py-5 ">
  <div class="container">
    <div class="row mb-5">
      <div class="col-lg-12">
        <p>
          Our informatics resources are freely-available, open-source tools that help research scientists from around
          the world identify and prioritise potential therapeutic drug targets. We maintain two 
          <a href="#flagship-resources">flagship resources</a> - 
          the Open Targets Platform and the Open Targets Genetics Portal. We also maintain a vareity of 
          <a href="#other-resources-and-tools">other specialised tools and resources</a>
          that tackle specific research questions.
        </p>
      </div>
    </div>
    <div class="row">
      <div class="col-lg-12" id="flagship-resources">
        <h3>Flagship resources</h3>
      </div>
    </div>
    <div class="row mb-5">
      <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12 py-3 d-flex">
        <div class="card p-4 rounded align-self-stretch">
          <img class="img-fluid mb-3" src="/assets/img/open-targets-platform-screenshot.jpg" />
          <h4 class="text-center mb-3">Open Targets Platform</h4>
          <p>
            The Open Targets Platform - previously called the Target Validation Platform - 
            is a freely available resource for the integration of genetics, omics and 
            chemical data to aid systematic drug target identification and prioritisation. 
            The Platform provides summaries of the evidence (e.g. germline variants, 
            somatic mutations, pathways, drugs) for the involvement of a specific gene with 
            a selected disease. It supports disease and target centric workflows, displays 
            the evidence for target-disease associations, as well as profiles of relevant 
            target and disease annotations. Please see
            <a href="https://academic.oup.com/nar/article/47/D1/D1056/5193331" target="_blank">Carvalho-Silva et al. 2019</a>
            for more information on the Platform features and scoring methodologies.
          </p>
          <a class="text-center" href="https://www.targetvalidation.org" target="_blank">
            <button class="btn btn-primary">Vist the Open Targets Platform</button>
          </a>
        </div>
      </div>      
      <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12 py-3 d-flex">
        <div class="card p-4 rounded align-self-stretch">
          <img class="img-fluid mb-3" src="/assets/img/open-targets-genetics-portal-screenshot.jpg" />
          <h4 class="text-center mb-3">Open Targets Genetics Portal</h4>
          <p>
            Open Targets Genetics is a portal for investigation of Genome Wide Association Study (GWAS) 
            data to assist in identifying the causal genes underlying each association, and hence to 
            prioritise drug targets. The portal aggregates and merges genetic associations curated from 
            literature and newly-derived loci from UK Biobank with (open source) functional genomics 
            data including epigenetics (e.g., chromatin conformation, chromatin interactions) and 
            quantitative trait loci (e.g., eQTLs from GTEX, pQTL), and applies statistical fine-mapping 
            across thousands of trait-associated loci, to resolve association signals and link each 
            variant to its proximal and distal target gene(s), using a single evidence score.
          </p>
          <a class="text-center" href="https://genetics.opentargets.org/" target="_blank">
            <button class="btn btn-primary">Vist the Open Targets Genetics Portal</button>
          </a>
        </div>
      </div>
    </div>
    <div class="row">
      <div class="col-lg-12" id="other-resources-and-tools">
        <h3>Other resources and tools</h3>
      </div>
    </div>
    <div class="row mb-5">
      <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12 py-3 d-flex">
        <div class="card p-4">
          <img class="img-fluid mb-3" src="/assets/img/project-score-screenshot.jpg" />
          <h4 class="text-center mb-3">Project Score</h4>
          <p>
            Project Score is a web portal that allows researchers to explore the results of CRISPR-Cas9 whole-genome 
            drop out screens across a diverse collection of human cancer cell models and to identify dependencies in 
            cancer cells to help guide precision cancer medicines. Please see 
            <a href="https://www.nature.com/articles/s41586-019-1103-9" target="_blank">Behan et al. 2019</a> for more information.
          </p>
          <a class="text-center" href="https://score.depmap.sanger.ac.uk/" target="_blank">
            <button class="btn btn-primary">Vist the Project Score website</button>
          </a>
        </div>
      </div>
      <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12 py-3 d-flex">
        <div class="card p-4">
          <img class="img-fluid mb-3" src="/assets/img/epichoose-screenshot.jpg" />
          <h4 class="text-center mb-3">epiChoose</h4>
          <p>
            epiChoose is an application for quantifying the relatedness between cell lines and primary cells. 
            We have extensively profiled a number of commonly used cell line models across a number of tissues. 
            This profiling consists of epigenetic (histone modification, CTCF, ATAC-seq) and transcriptional 
            (RNA-seq) whole-genome measurements. From this data, we have established a platform that provides 
            information on the “distance” between candidate cell models and target primary cells. By ranking 
            and selecting cell line models in a data-driven manner, the best model for the experiment 
            can be chosen, based on epigenetic and transcriptional evidence and not historical usage.
          </p>
          <a class="text-center" href="http://epichoose.shiny.opentargets.io/epiChoose/" target="_blank">
            <button class="btn btn-primary">Vist the epiChoose tool page</button>
          </a>
        </div>
      </div>
      <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12 py-3 d-flex">
        <div class="card p-4">
          <img class="img-fluid mb-3" src="/assets/img/cellector-screenshot.jpg" />
          <h4 class="text-center mb-3">CELLector</h4>
          <p>
            CELLector is a computational tool implemented in an open source R Shiny application and R package that 
            allows researchers to select the most relevant cancer cell lines in a genomic-guided fashion. 
            CELLector combines methods from graph theory and market basket analysis; it leverages tumour genomics 
            data to explore, rank, and select optimal cell line models in a user-friendly way, enabling scientists 
            to make appropriate and informed choices about model inclusion/exclusion in retrospective analyses and 
            future studies. Additionally, it allows the selection of models within user-defined contexts, for 
            example, by focusing on genomic alterations occurring in biological pathways of interest or considering 
            only predetermined sub-cohorts of cancer patients. Finally, CELLector identifies combinations of 
            molecular alterations underlying disease subtypes currently lacking representative cell lines, providing 
            guidance for the future development of new cancer models. Please see 
            <a href="https://www.biorxiv.org/content/early/2018/03/03/275032" target="_blank">Najgebauer et al. 2018</a> 
            for an explanation of the approach.
          </p>
          <a class="text-center" href="http://ot-cellector.shiny.opentargets.io/CELLector_App/" target="_blank">
            <button class="btn btn-primary">Vist the CELLector tool page</button>
          </a>
        </div>
      </div>
      <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12 py-3 d-flex">
        <div class="card p-4">
          <img class="img-fluid mb-3" src="/assets/img/epichoose-screenshot.jpg" />
          <h4 class="text-center mb-3">DoRothEA</h4>
          <p>
            DoRothEA (Discriminant Regulon Expression Analysis) is a research resource that can be used to search 
            candidate TF-drug interactions in cancer. Due to their location as downstream effectors of signalling pathways, 
            aberrant activities in upstream driver genes (even if not mutated) will cause altered TF activities, thus 
            proposing TFs as sensors of pathway dysregulation and alternative markers. Here we study the role of 127 TFs in 
            drug sensitivity across ~1,000 cancer cell lines screened with 265 anti-cancer compounds from the GDSC. In our 
            first approach we studied how the TF activity pattern of an individual affects drug response and mined for single 
            TF-drug statistical interactions. In our second approach we screened for TFs whose activity patterns complement 
            or improve well-established genomic markers in the prediction of drug response. Please see 
            <a href="http://cancerres.aacrjournals.org/content/early/2017/12/09/0008-5472.CAN-17-1679" target="_blank">Garcia-Alonso et al. 2017</a> 
            for an explanation of the approach.
          </p>
          <a class="text-center" href="http://dorothea.opentargets.io/#/" target="_blank">
            <button class="btn btn-primary">Vist the DoRothEA website</button>
          </a>
        </div>
      </div>
      <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12 py-3 d-flex">
        <div class="card p-4">
          <img class="img-fluid mb-3" src="/assets/img/link-screenshot.png" />
          <h4 class="text-center mb-3">LINK</h4>
          <p>
            A new tool called LINK (LIterature coNcept Knowledgebase) has been developed that allows the exploration of 
            half a billion relations between genes, diseases, drugs and key concepts extracted from PubMed abstracts using 
            NLP (Natural Language Processing). Once MEDLINE relaxed their license for obtaining and analysing publication data,
            we started looking for novel ways to mine their data. We wanted to exploit the biomedical knowledge often buried 
            in the literature to help scientists generate new hypotheses for the identification of new drug targets. For this 
            purpose, we have built Library, an open source ecosystem comprising: a pipeline that allows us to quickly run a
            large-scale NLP analysis; an API that serves the resulting data; and a user interface to explore the data. Our 
            pipeline annotates genes, diseases and drugs present in PubMed abstracts, and extracts key concepts. Please see
            <a href="https://blog.opentargets.org/2018/01/18/link/" target="_blank">Andrea Pierleoni’s blog post</a> for more information.
          </p>
          <a class="text-center" href="https://link.opentargets.io/" target="_blank">
            <button class="btn btn-primary">Vist the LINK website</button>
          </a>
        </div>
      </div>
      <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12 py-3 d-flex">
        <div class="card p-4">
          <img class="img-fluid mb-3" src="/assets/img/crispr-cleanr-screenshot.png" />
          <h4 class="text-center mb-3">CRISPRcleanR</h4>
          <p>
            Genome editing by CRISPR-Cas9 technology allows large-scale screening of gene essentiality in cancer. 
          A confounding factor when interpreting CRISPR-Cas9 screens is the high false-positive rate in detecting 
          essential genes within copy number amplified regions of the genome. We have developed the computational 
          tool CRISPRcleanR which is capable of identifying and correcting gene-independent responses to CRISPR-Cas9 
          targeting. CRISPRcleanR uses an unsupervised approach based on the segmentation of single-guide RNA fold 
          change values across the genome, without making any assumption about the copy number status of the 
          targeted genes. CRISPRcleanR is implemented as an R package and as an interactive Python package with 
          full documentation, tutorials, built in datasets to reproduce the results in this manuscript, and is 
          publically available as an <a href="https://github.com/francescojm/CRISPRcleanR" target="_blank">R package</a> 
          and as a <a href="https://github.com/cancerit/pyCRISPRcleanR" target="_blank">Python package</a>. 
          The Python implementation is dockerised making it platform-independent and 
          <a href="https://dockstore.org/containers/quay.io/wtsicgp/dockstore-pycrisprcleanr" target="_blank">usable in cloud environments</a>.
          Please see <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-4989-y" target="_blank">Iorio et al. 2018</a> 
          for more information.
          </p>
          <a class="text-center" href="https://github.com/francescojm/CRISPRcleanR" target="_blank">
            <button class="btn btn-primary">Vist the CRISPRcleanR GitHub repo</button>
          </a>
        </div>
      </div>
    </div>
  </div>
</section>



