---
layout: default
permalink: /science
title: Science
---

<header class="py-5" style="background-color:#3489ca">
  <div class="container">
    <div class="row">
      <div class="col-sm-12 text-center text-white">
        <h1>{{ page.title }}</h1>
      </div>
    </div>
  </div>
</header>

<section class="py-5 my-5">
  <div class="container">
    <div class="row">
      <div class="col-lg-12">
        <h2>Scientific overview</h2>
        <p>
          At Open Targets, we aim to improve how drug targets are selected for target-based drug discovery. 
          We recognise that genetically validated targets are more likely to progress to clinical trials 
          and that the influx of new data from modern genetics and genomics allows targets to be considered 
          in a genome-wide comprehensive context. In addition, new technologies in gene editing, single 
          cell sequencing and cell models allow new types of data to be generated to address target selection.
        </p>
        <p>
          We consider target selection to be composed of a target identification step, where targets with 
          significant associations to the disease biology are identified, and a prioritisation step where the 
          list of targets is refined both on the strength of the association evidence and the additional target 
          and disease parameters such as target tractability by drug modality, and likely target based safety 
          risks. We identify areas to work in that will enhance the process of target identification and 
          prioritisation either by integrating and analysing data, or by generating new data in our therapeutic 
          areas of interest.
        </p> 
        <p>
          We place in the public domain all our new informatics tools, experimental methods, platforms and the 
          data generated by our projects as soon as is practical. We do not plan to seek patent protection for IP 
          arising from Open Targets. However, we recognise that instances may arise where this is appropriate to 
          support our mission. Therefore, Open Targets has a Joint Patent Committee with scientific and legal 
          experts from all of our partners that reviews all potential publications (but not raw data) prior to 
          submission. 
        </p>
        <p>
          In practice, this means:
        </p>
        <ul>
          <li>
            For the development and execution of Open Targets Research Projects each Member has agreed to license 
            its Background IP to the other members.
          </li>
          <li>
            All Members have agreed to license Open Targets Arising IP to each other for use in Open Targets Research 
            Projects and for the Members’ research and development activities.
          </li>
          <li>
            Any IP that relates directly and solely to an Industry Partner Compound will belong to the Industry Partner.
          </li>
          <li>
            Any other IP arising from an Open Targets Research Project will belong to the Member(s) that invented it.
          </li>
        </ul>
      </div>
    </div>
  </div>
</section>

<section class="py-5 my-5" style="background-color: lightgrey;">
  <div class="container">
    <div class="row">
      <div class="col-lg-12">
        <h2>Experimental projects</h2>
        <p>
          The Open Targets portfolio of experimental projects is designed to provide new information in our key therapy 
          areas to enable target identification and prioritisation. We perform high throughput experimental projects that 
          generate target-centred data in human, physiologically relevant systems to improve the strength of causal links 
          between targets and diseases in our focus therapeutic areas of Oncology, Immunity and Inflammation and 
          Neurodegeneration. We use high throughput methods to address as near as possible the full range of relevant 
          targets (ideally the whole genome) in systems that are most relevant to the disease.  We use a number the 
          expertise of our partners in emerging and established technologies such as:
        </p>
        <ul class="mb-5">
          <li>
            Gene editing (CRISPR)
          </li>
          <li>
            Induced pluripotent stem cells
          </li>
          <li>
            Single cell genomics
          </li>
          <li>
            Organoid and tissue culture
          </li>
          <li>
            Large-scale genomics and epigenomics (e.g. ENCODE)
          </li>
          <li>
            Genome-wide sequencing studies
          </li>
          <li>
            Next-generation sequencing
          </li>
        </ul>
        <h4>Oncology</h4>
        <p class="mb-5">
          Our projects in Oncology are connect to resources and expertise within the Sanger Institute's cancer program, 
          which has played an important role in understanding the genetic basis of cancer. A shared theme is the use of 
          accessible cancer resources to curate and analyse clinical genomic datasets to identify driver genes 
          (mutations, amplification, deletions and 
          <a href="https://www.nature.com/articles/s41467-019-09940-1" target="_blank">gene fusions</a>
          across multiple cancer sub-types. A key resource is the unique collection of >1000 human cancer cell lines at 
          the Sanger Institute along with their drug sensitivities. Genomic information including RNA-seq and synthetic 
          lethality from genome editing in model systems that best reflect the biology of tumours identifies many new 
          target opportunities as exemplified in <a href="https://www.nature.com/articles/s41586-019-1103-9" target="_blank">
          Behan et al 2019</a>. We also are using co-culture approaches to reveal new immune-oncology targets
        </p>
        <h4>Immunology</h4>
        <p class="mb-5">
          We have focussed on explaining the genetic basis of a number of autoimmune diseases including inflammatory bowel 
          disease (IBD) where there is both a strong interest from our partners and substantial expertise on the Genome Campus. 
          We have developed a state of the art meta-analysis for the existing IBD cohorts, and will move candidate targets 
          from this into genome-wide knockouts in gut epithelium organoids for validation. In addition we are generating 
          eQTL datasets that can be used to resolve the causal genes at loci implicated from genetics. As Open Targets has 
          progressed we have expanded to additional projects probing the role of targets either in well-defined immune cells 
          through gene editing and single cell transcriptoics and epigenetic profiling (for instance in macrophages, 
          dendritic cells and 
          <a href="https://www.nature.com/articles/s41588-019-0493-9" target="_blank">T cells in response to various stimulations</a>, 
          or in disease such as asthma using <a href="https://www.nature.com/articles/s41591-019-0468-5" target="_blank">single cell genomics</a>. 
          A project to identify receptor ligand pairs in NK cells with application in immune-oncology crosses these two therapeutic areas.
        </p>
        <h4>Neurodegeneration</h4>
        <p>
          In Neurodegeneration we have used the collective expertise of our partners and through collaboration with the Gurdon 
          Institute built a set of projects based around Alzheimer’s and Parkinson’s disease. These projects use similar approaches 
          to those described above, such as gene editing in neurons derived from iPS cells to identify modifiers of the response 
          to oxidative stress, mechanism of Tau uptake and the effects of Alzheimer’s disease specific mutations. We are using 
          fine mapping of GWAS in Alzheimer’s and Parkinson’s disease to identify and test potential targets in the same neuron 
          systems and characterising these systems at the single cell genomics level. 
        </p>
      </div>
    </div>
  </div>
</section>

<section class="py-5 my-5">
  <div class="container">
    <div class="row">
      <div class="col-lg-12">
        <h2>Core bioinformatics and data generation pipelines</h2>
        <p class="mb-5">
          The big challenge for our core bioinformatics work is how to bring together the 
          various, relevant data on targets in a way that allows simple & seamless exploration 
          by drug discovery scientists. We aim to cover many data types relevant to human 
          disease biology using the expertise from Open Targets partners and beyond.
        </p>
        <h4>Open Targets Platform</h4>
        <p class="mb-5">
          Our overall approach to target identification and prioritisation is crystallised in the 
          <a href="https://www.targetvalidation.org" target="_blank">Open Targets Platform</a>, which integrates public data 
          relevant to the association between targets and diseases, and provides additional data and tools for prioritisation. 
          The Platform was developed together with scientists from our partners through a user experience (UX) design process. 
          We integrate data from genetics, somatic mutations, expression analysis, drugs, animal models and the literature 
          through robust pipelines and assess the association of a target with disease in a single score. User friendly 
          interfaces are used to guide the user in target identification and allowing access to the evidence. In addition 
          prioritisation information is provided in the form of tractability assessment of the target, known safety liabilities, 
          gene expression information and other target properties. The Platform is updated with a new release every 2 months in 
          in the cloud with separate geolocated instances, is accessible via an application programming interface 
          (API) with <a href="https://docs.targetvalidation.org/programmatic-access/api-tutorials" target="_blank">tutorials</a>, 
          and is developed as an open source project on <a href="https://github.com/opentargets/platform" target="_blank">Github</a>.
        </p>
        <h4>Open Targets Genetics Portal</h4>
        <p class="mb-5">
          Given the enhanced success of genetically validated targets in clinical trial progression, we wish to enhance the use of 
          human genetics information in target selection. We have developed the 
          <a href="https://genetics.opentargets.org/" target="_blank">Open Targets Genetics Portal</a> to provide a way for users 
          to investigate Genome Wide Association Study (GWAS) data to assist in identifying the causal genes underlying each association 
          and to prioritise drug targets. The Portal was initially at the American Society of Human Genetics conference in October 2018. 
          It supports searches that start with a gene, a single variant, a single study (trait), or multiple studies and 
          users can also <a href="https://genetics-docs.opentargets.org/technical-pipeline/data-download" target="_blank">download the data</a>
          generated for each Portal release. Collaborative work with the <a href="https://www.ebi.ac.uk/gwas/" target="_blank">GWAS catalog</a> 
          has allowed us to include data from genome-wise summary statistics including UK Biobank. The Portal now also includes workflows to 
          identify candidate causal variants and to analyse colocalization of GWAS and eQTL signals to assist in identifying causal genes. 
        </p>
        <h4>Other bioinformatics services and tools</h4>
        <p>          
          Additional ancillary projects in the area of bioinformatics include development of an 
          <a href="https://ensembl.github.io/eQTL-Catalogue-website/" target="_blank">eQTL database</a>, network analysis for drug target list 
          expansion, and enhanced data for the Open Targets Platform, including additional data from clinical trial records and data on the 
          effect of mutations on protein function. 
            
        </p>
      </div>
    </div>
  </div>
</section>